client/app/directives/resize-event.js

```diff
@@ -4,20 +4,22 @@ const items = new Map();
 
 function checkItems() {
   items.forEach((item, node) => {
-    const offsetWidth = node.offsetWidth;
-    const offsetHeight = node.offsetHeight;
+    const bounds = node.getBoundingClientRect();
+    // convert to int (because these numbers needed for comparisons), but preserve 1 decimal point
+    const width = Math.round(bounds.width * 10);
+    const height = Math.round(bounds.height * 10);
```

Comment:
This definitely works but kinda confusing cause it's not storing the actual dimensions.
Perhaps `bounds.width.toFixed(1)` can work?

Reply:
Well, it's actually the same - just I store `int`, but this will store `string`. But in fact we just need some normalized values that we can compare. Since your variant will do exactly the same as mine, and we'll remove this directive after React migration - I prefer to keep this as is.